.github/workflows/rust.yaml

```diff
@@ -1,11 +1,17 @@
 name: Rust CI
 on:
   pull_request:
+    branches:
```

Comment:
this is not needed

Reply:
branches is required. `paths` is not needed right?

Reply:
It's not required, you see, it's running today with the checked in code right?